unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDController.cs

```diff
@@ -512,7 +574,12 @@ public void RandomizeWearables()
         EquipHairColor(hairColorList.colors[Random.Range(0, hairColorList.colors.Count)]);
         EquipEyesColor(eyeColorList.colors[Random.Range(0, eyeColorList.colors.Count)]);
 
-        model.wearables.Clear();
+        List<WearableItem> wearablesToRemove = model.wearables.Where(x => !x.IsEmote()).ToList();
+        foreach (var wearable in wearablesToRemove)
+        {
+            model.wearables.Remove(wearable);
+        }
```

Comment:
should we also keep the `model.wearables.Clear();` after calling the `.Remove()` on all those wearables?